### DISCUSSION TITLE
docs

### DISCUSSION THREAD

**Discussion by pizzav-xyz** [modrinth/.github#153] [Other discussions]
can you add docs on how to install modpacks

**Comment by triphora** [comment]
Maybe try checking the docs first to see that such a thing already exists. https://docs.modrinth.com/docs/modpacks/playing_modpacks/

**Reply by pizzav-xyz** [reply]
not for modrinth app only for at launcher prism launcher mutlimc and BakaXL

**Reply by triphora** [reply]
Seems fairly self-explanatory to me—you go to browse for modpacks and you just hit install. However, I guess such a thing would still be worth noting on that page, if you could open an issue on the docs repo.
